Updated to rattler-build setup

darynwhite opened the pull request:
…requirements.

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->
This may also solve the issues found in #52

darynwhite committed:
Updated to rattler-build setup. Attempting to account for dependency requirements.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found some lint.

Here's what I've got...


For **recipe/recipe.yaml**:

 * ❌ `noarch` packages can't have skips with selectors. If the selectors are necessary, please remove `noarch: python`.

For **recipe/recipe.yaml**:

 * ℹ️ `noarch: python` recipes should usually follow the syntax in our [documentation](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python) for specifying the Python version.
   - For the `host` section of the recipe, you should usually use the pin python ${{ python_min }}.* for the `python` entry.
   - For the `tests[].python.python_version` or `tests[].requirements.run` section of the recipe, you should usually use the pin `python_version: ${{ python_min }}.*` or `python ${{ python_min }}.*` for the `python_version` or `python` entry.
   - If the package requires a newer Python version than the currently supported minimum version on `conda-forge`, you can override the `python_min` variable by adding a Jinja2 `set` statement at the top of your recipe (or using an equivalent `context` variable for v1 recipes).


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18386502510](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18386502510). Examine the logs at this URL for more detail.</sub>

darynwhite committed:
Refactor requirements: simplify pandas and numpy version constraints; update license file path

darynwhite committed:
Add build_config

darynwhite committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.08.21.44.29

Other tools:
- conda-build 25.9.0
- rattler-build 0.48.1
- rattler-build-conda-compat 1.4.9

darynwhite committed:
Correctly use `python_min`

darynwhite committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.08.21.44.29

Other tools:
- conda-build 25.9.0
- rattler-build 0.48.1
- rattler-build-conda-compat 1.4.9

darynwhite committed:
Remove build skip

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/recipe.yaml**:

 * ℹ️ `noarch: python` recipes should usually follow the syntax in our [documentation](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python) for specifying the Python version.
   - For the `host` section of the recipe, you should usually use the pin python ${{ python_min }}.* for the `python` entry.
   - For the `tests[].python.python_version` or `tests[].requirements.run` section of the recipe, you should usually use the pin `python_version: ${{ python_min }}.*` or `python ${{ python_min }}.*` for the `python_version` or `python` entry.
   - If the package requires a newer Python version than the currently supported minimum version on `conda-forge`, you can override the `python_min` variable by adding a Jinja2 `set` statement at the top of your recipe (or using an equivalent `context` variable for v1 recipes).


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18386543047](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18386543047). Examine the logs at this URL for more detail.</sub>

thewchan commented:
This looks good I just need to adjust to address the lint messages

thewchan committed:
bump build num + address lint message

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

thewchan commented:
@darynwhite LGTM
